docs/blog/posts/2021-05-06-tf.md

```diff
@@ -18,3 +20,5 @@ These builds provide a significant advantage as illustrated below over the stand
 ![Benchmark image](../../assets/images/gpu_ai_benchmark.png)
 
 A recent tutorial was given on this topic, for more information see the [recording](https://web.microsoftstream.com/video/af9b54ae-9158-4075-9f36-9aa2a4412ad0) or checkout the [tutorial materials](https://github.com/NREL/HPC/tree/master/workshops/Optimized_TF)
+
+...
```

Comment:
Do we need ellipses at the end of the posts?

Reply:
They are not. I'm going to remove them in the next commit.